src/client/entity-editor/identifier-editor/reducer.js

```diff
@@ -53,7 +53,10 @@ function reducer(
 			return state.setIn([payload.rowId, 'type'], payload.value);
 		case REMOVE_IDENTIFIER_ROW:
 			return state.delete(payload);
-
+		case REMOVE_EMPTY_IDENTIFIERS:
+			return state.filterNot(identifier =>
+				identifier.get('value') === '' || identifier.get('type') === null
```

Comment:
Hi @Ge0rg3 !

I was able to test locally, and I see everything is working fine. The code is also clean and readable, everything in the right place. Well done !

I have one suggestion, after some consideration: removing identifiers only when they have both value **and** type empty.
The reason for that is that we currently have a validation system that will show users if they only entered one. For example, a user might paste in a value but forget to select a type (or think it was auto-detected). In that case, the validator will show an error, and the user can correct the identifier rather than have to start from scratch, wondering where the previous one went.

Whereas identifiers that are totally empty, we know we can get rid of as we're not destroying any information they could store.

I hope I'm making sense.
Thanks for sticking to it, I'm looking forward to merging the PR, it's a good improvement!